GOVERNANCE.md

```diff
@@ -1,3 +1,175 @@
-This is a placeholder governance file for the chapel-lang organization.
+Governance of the Chapel Project
+================================
 
-TODO: Complete its contents
\ No newline at end of file
+This file describes the current governance of the Chapel Language
+project, covering both the language's definition and the code base
+that implements its compiler, library modules, and runtime.  It was
+last updated in January 2025.
+
+Over time, we anticipate improving the Chapel project's governance,
+with the goal of making it more of a community-governed project.  If
+this topic is of interest to you, or you would like to play a role in
+that transition or its result, please let us know.  If you have
+specific requests with respect to how the project is governed, please
+feel encouraged to capture those in an issue on this repository, or to
+[reach out to the community](https://chapel-lang.org/community/).
+
+
+Technical Leadership
+--------------------
+
+**Technical Lead:** [Brad Chamberlain](https://github.com/bradcray)
+
+Role: Generally speaking, the role of the project technical lead has
+been to help guide decisions for the project in ways that are
+technically sound, responsive to user needs and concerns, cognizant of
+the project's history and past decisions, and compatible with other
+efforts being done across the project.
+
+As a founding member of the Chapel project who has been continually
+involved with it since the outset, Brad has served in this role since
+2006.  In practice, he serves as something of a
+[BDFL](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life)
+role for the project to date.
+
+
+**Subteam Leads**
+
+Role: As the scope of the Chapel project has grown, a number of
+subteams have been spun up to focus on specific initiatives at finer
+granularities.  Generally speaking, these subteam leads have a great
+deal of latitude in making decisions to meet their goals, but are
+expected to keep the technical lead aware of major decisions,
+particularly for those that are contentious, represent a significant
+change in direction, or have the possibility of impacting other
+subteams.
+
+At present, the Chapel subteam leads are as follows:
+
+* Community Management: [Brandon Neth](https://github.com/brandon-neth)
+* Compiler: [Ben Harshbarger](https://github.com/benharsh)
+* DevOps: [Tim Zinsky](https://github.com/tzinsky)
+* GPU: [Engin Kayraklioglu](https://github.com/e-kayrakli)
+* Performance: [Andy Stone](https://github.com/stonea)
+* Runtime / Platform / Portability: [John Hartman](https://github.com/jhh67)
+* User Support / Language Stabilization: [Lydia Duncan](https://github.com/lydia-duncan)
+
+
+Core Contributors
```

Comment:
What if we just called this section / these people "Committers" ? I suppose that might make the Decision-making section weird, but maybe not?